packages/indicators/src/builders/buildArithmetic/buildArithmetic.ts

```diff
@@ -40,14 +40,16 @@ const fetchAnalyticClusters = async (
   return clusters.map(replaceAnalyticValuesWithDefaults).filter(checkClusterIncludesAllElements);
 };
 
-const buildAnalyticValues = (analyticClusters: AnalyticCluster[], formula: string) =>
-  analyticClusters
-    .map(({ organisationUnit, period, dataValues }) => ({
-      organisationUnit,
-      period,
-      value: runArithmetic(formula, dataValues),
-    }))
+const buildAnalyticValues = (analyticClusters: AnalyticCluster[], formula: string) => {
+  const parser = new ExpressionParser();
+
+  return analyticClusters
+    .map(({ organisationUnit, period, dataValues }) => {
+      parser.setScope(dataValues);
```

Comment:
`parser.setScope()` will just keep setting new values into the scope without clearing the previous values. Do you think there might be any side effects if we use the same parser (which will share same scope) to evaluate the formula for multiple analytic clusters? Or is it safer to maybe instantiate new parser in the for loop?